fakestorage/server.go

```diff
@@ -220,6 +220,11 @@ func (s *Server) buildMuxer() {
 		r.Path("/b/{bucketName}/o/{objectName:.+}").Methods(http.MethodPut, http.MethodPost).HandlerFunc(jsonToHTTPHandler(s.updateObject))
 	}
 
+	// Internal / update server configuration
+	s.mux.Path("/internal/config").Methods(http.MethodPut).HandlerFunc(jsonToHTTPHandler(s.updateServerConfig))
```

Comment:
nit: can we prefix it with an `_`. Not that relevant, just to make the intent even more explicit.
```suggestion
	s.mux.Path("/_internal/config").Methods(http.MethodPut).HandlerFunc(jsonToHTTPHandler(s.updateServerConfig))
```

Reply:
looks good to me, renamed